libcxx/cmake/caches/hexagon-linux-runtimes.cmake

```diff
@@ -0,0 +1,25 @@
+
+set(CMAKE_EXE_LINKER_FLAGS "-lclang_rt.builtins-hexagon -nostdlib" CACHE STRING "")
+set(CMAKE_SHARED_LINKER_FLAGS "-lclang_rt.builtins-hexagon -nostdlib" CACHE STRING "")
+set(CMAKE_CXX_COMPILER_TARGET hexagon-unknown-linux-musl CACHE STRING "")
+
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR OFF CACHE BOOL "")
+set(LLVM_ENABLE_RUNTIMES "libcxx;libcxxabi;libunwind;compiler-rt" CACHE STRING "")
+set(LIBCXX_INCLUDE_BENCHMARKS OFF CACHE BOOL "")
+set(LIBCXX_HAS_MUSL_LIBC ON CACHE BOOL "")
+set(LIBCXX_INCLUDE_TESTS ON CACHE BOOL "")
+set(LIBCXXABI_INCLUDE_TESTS ON CACHE BOOL "")
+set(LIBCXX_CXX_ABI libcxxabi CACHE STRING "")
+set(LIBCXXABI_USE_LLVM_UNWINDER ON CACHE BOOL "")
+set(LIBCXXABI_HAS_CXA_THREAD_ATEXIT_IMPL OFF CACHE BOOL "")
+set(LIBCXXABI_ENABLE_SHARED ON CACHE BOOL "")
+set(COMPILER_RT_DEFAULT_TARGET_TRIPLE hexagon-unknown-linux-musl CACHE STRING "")
+set(COMPILER_RT_BUILD_BUILTINS OFF CACHE BOOL "")
+set(COMPILER_RT_BUILD_SANITIZERS ON CACHE BOOL "")
+set(COMPILER_RT_SUPPORTED_ARCH hexagon CACHE STRING "")
+set(COMPILER_RT_USE_LLVM_UNWINDER ON CACHE BOOL "")
```

Comment:
I don't think those settings belong in this cache?

Reply:
> I don't think those settings belong in this cache?

This comment refers to all five of those?  `COMPILER_RT_DEFAULT_TARGET_TRIPLE`, `COMPILER_RT_BUILD_BUILTINS`, `COMPILER_RT_BUILD_SANITIZERS`, `COMPILER_RT_SUPPORTED_ARCH`, `COMPILER_RT_USE_LLVM_UNWINDER`?

Of those,  `COMPILER_RT_BUILD_BUILTINS`, `COMPILER_RT_BUILD_SANITIZERS` are `option()`s used to control the build scope - those can stay, right?  And `COMPILER_RT_USE_LLVM_UNWINDER` - this is an `option()` too.  Can't we set this here?

`COMPILER_RT_SUPPORTED_ARCH`: this has been removed.

`COMPILER_RT_DEFAULT_TARGET_TRIPLE` is still present, but if it's not allowed/expected I will explore removing it.  It may be redundant with `LLVM_TARGET_TRIPLE`?